src/app/app.component.html

```diff
@@ -0,0 +1,20 @@
+<!--The content below is only a placeholder and can be replaced.-->
```

Comment:
what is the need of  this file and its html tag ?

Reply:
ok sir remove this this